Example/PaymentSheet Example/PaymentSheet Example.xcodeproj/project.pbxproj

```diff
@@ -593,6 +596,7 @@
 				84CFB1ACA4915EE5B2E25341 /* EndpointSelectorViewController.swift in Sources */,
 				EB3C76C2A5704F94F0414BEF /* ExampleCheckoutViewController.swift in Sources */,
 				D235E82A7173E8051BC5A261 /* ExampleCustomCheckoutViewController.swift in Sources */,
+				615739F129D218C700111D23 /* ExampleCustomDeferredCheckoutViewController.swift in Sources */,
```

Comment:
@eurias-stripe I'm confused here, I thought we removed project files?

Reply:
Is this because we just did a release and I picked up these before they could be removed?